kde-apps/dolphin/patches/dolphin-24.12.3.patchset

```diff
@@ -0,0 +1,65 @@
+From 61b9e99a641fe2e33010d1d5e5ad67cf6606ffdb Mon Sep 17 00:00:00 2001
+From: Schrijvers Luc <begasus@gmail.com>
+Date: Tue, 16 Jul 2024 17:19:06 +0200
+Subject: Build fix
+
+
+diff --git a/src/panels/information/informationpanel.cpp b/src/panels/information/informationpanel.cpp
+index 02fe3e3..63303a2 100644
+--- a/src/panels/information/informationpanel.cpp
++++ b/src/panels/information/informationpanel.cpp
+@@ -8,7 +8,9 @@
+ 
+ #include "informationpanelcontent.h"
+ 
+-#include <KDirNotify>
++#ifndef __HAIKU__
++    #include <KDirNotify>
++#endif
+ #include <KIO/StatJob>
+ #include <KJobWidgets>
+ #include <KLocalizedString>
+@@ -406,6 +408,7 @@ void InformationPanel::init()
+     Q_ASSERT(m_urlChangedTimer->interval() < m_infoTimer->interval());
+     Q_ASSERT(m_urlChangedTimer->interval() < m_resetUrlTimer->interval());
+ 
++#ifndef __HAIKU__
+     org::kde::KDirNotify *dirNotify = new org::kde::KDirNotify(QString(), QString(), QDBusConnection::sessionBus(), this);
+     connect(dirNotify, &OrgKdeKDirNotifyInterface::FileRenamed, this, &InformationPanel::slotFileRenamed);
+     connect(dirNotify, &OrgKdeKDirNotifyInterface::FilesAdded, this, &InformationPanel::slotFilesAdded);
+@@ -413,6 +416,7 @@ void InformationPanel::init()
+     connect(dirNotify, &OrgKdeKDirNotifyInterface::FilesRemoved, this, &InformationPanel::slotFilesRemoved);
+     connect(dirNotify, &OrgKdeKDirNotifyInterface::enteredDirectory, this, &InformationPanel::slotEnteredDirectory);
+     connect(dirNotify, &OrgKdeKDirNotifyInterface::leftDirectory, this, &InformationPanel::slotLeftDirectory);
++#endif
```

Comment:
I guess this would need a custom implementation on Haiku? (probably based on NodeMonitor)

Reply:
Got this in since a few versions, iirc I got an error on it not being defined. There isn't a qml file fot it.

Reply:
Narrowed it down to KIO/KJobWidgets, will do some more checking on this.

Reply:
Upstreamed patch at: https://invent.kde.org/system/dolphin/-/merge_requests/936 :)